src/styles/common/_mixins.scss

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ @import "./../common/colors";
+ @import "./../common/layout";
+ @import "./../common/borders";
+ @import "./../common/fonts";
+
+
+ @mixin link_button {
```

Comment:
Please add documentation for each mixin, when to use each mixin

Reply:
Done